recipes/chfem/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "chfem" %}
+{% set version = "2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/cortezpedro/chfem/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 492acd0561054c5bdecc6b2f2694df9d32b430c784f6715a64722af9ea58ec3d
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [osx]
+  skip: true  # [cuda_compiler_version in (undefined, "None")]
+  missing_dso_whitelist:
+    - "*/nvcuda.dll"   # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - libgomp  # [linux]
+    - llvm-openmp  # [win]
+    - python
```

Comment:
Would drop `python` from `requirements/build`. This is only for build tools (used via CLI and not linked to). For `python` it should be in `requirements/host` & `requirements/run`

```suggestion
```

Reply:
Thank you for your comments! I'll try that right now. However I'm not sure that it will resolve the problem I mentioned in the chat. It is strange because it looks like the package gets built 4 times (as seen [here](https://dev.azure.com/conda-forge/84710dde-1620-425b-80d0-4cf5baca359d/_apis/build/builds/899173/logs/59)). The first three times it compiles fine whereas the fourth time it encounters that crypy.h problem

Reply:
@jakirkham any other idea?